### PR TITLE
CLC-7515, roster photos: sections dropdown and student listing

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-7515

### CODE REVIEW

File: src/views/Roster.vue [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  mixins: [Context, Util],
+  components: {RosterPhoto},
+  data: () => ({
+    context: 'canvas',
+    courseId: undefined,
+    roster: undefined,
+    section: null
+  }),
+  mounted() {
+    // this.courseId = this.$route.query.id || 1493973
```
Comment (paulkerschen) [inline src/views/Roster.vue:108]: @johncrossman Stray debugging line?

Comment (johncrossman) [reply src/views/Roster.vue:108]: Ah yes, removing.

Comment (paulkerschen) [reply src/views/Roster.vue]: FYI, if we want to preserve the current routing in https://github.com/ets-berkeley-edu/calcentral/blob/master/src/assets/javascripts/angular/configuration/routeConfiguration.js, then the non-LTI route for this view would be `canvas/rosters/:canvas_course_id`. That way you can inject any course id you like during development.

Comment (johncrossman) [reply src/views/Roster.vue]: @paulkerschen Did you see the change in `router.ts`?  What you describe was my intention.

Comment (paulkerschen) [reply src/views/Roster.vue]: Oh yeah, there it is. I thought it wasn’t there before, but clearly I’m senile.

**Pull request merged**